### PR TITLE
Fix OpenGL extensions not loading

### PR DESCRIPTION
**Pull request by eXpl0it3r**
## Description

Fix OpenGL extensions not loading in a specific Linux VM configuration ([here](https://en.sfml-dev.org/forums/index.php?topic=21384.msg165289)).

Rebased the PR #1495

### CODE REVIEW

File: src/SFML/Window/GlContext.cpp [hunk 1]
```diff
@@ -234,28 +236,21 @@ namespace
                     extensionString++;
 
                 extensions.push_back(std::string(extension, extensionString));
-            }
-            while (*extensionString++);
+            } while (*extensionString++);
```
Comment (binary1248) [inline src/SFML/Window/GlContext.cpp]: Unnecessary change?

Comment (eXpl0it3r) [reply src/SFML/Window/GlContext.cpp]: Woops, oversight while resolving merge conflict.
